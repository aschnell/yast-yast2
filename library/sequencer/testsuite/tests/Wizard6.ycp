{

integer cur = 0;
list clicks = [ `next, `back, `back ];
include "./tests/Wizard.yh";

}
