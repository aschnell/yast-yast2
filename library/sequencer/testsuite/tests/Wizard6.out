Log	next
Log	back
Log	back
Return	`back
